src/naive/datetime/mod.rs

```diff
@@ -127,6 +127,28 @@ impl NaiveDateTime {
         datetime.expect("invalid or out-of-range datetime")
     }
 
+    /// Makes a new `NaiveDateTime` corresponding to a UTC date and time,
```

Comment:
You probably copy-pasted this from one of the other methods, but we're trying to make the first line of docstrings self-contained sentence fragments. After that we should have an empty line, then a more detailed description. Would you mind tweaking the docs for this new method at least?

Reply:
I think the whole doc can be simplified. Gave it a try. Please have a look.